### PR TITLE
✨ Add a way to customize applink scheme

### PR DESCRIPTION
**Pull request by istornz**
This feature intent to add a way to customize app link scheme.
By default value are set to "la" as seen in the documentation.

PR related to #41

### CODE REVIEW

**Comment by ggirotto** [comment]
@istornz Just a question: using `la` as the default value for the scheme wouldn't produce the issue you described [here](https://github.com/istornz/flutter_live_activities/pull/41#issuecomment-1770865805)? If we use as an assumption that most of the plugin users don't implement this Plugin <-> Flutter communication (which I personally think are most of the cases), all of these will use the same schema in the end.

TLDR: shouldn't `la` schema be or custom or `null` instead providing a default value?

**Comment by istornz** [comment]
@ggirotto Yes you'r right, I implemented a default value because on the doc, I write "la" & I'm scary about user's who forgot to change on Xcode & on the init ...

But I think I will let the scheme null & precise on the doc how to change it 👍
